### DISCUSSION TITLE
About Layout Layers (In the guide)

### DISCUSSION THREAD

**Discussion by GronbergInfo** [Textualize/textual#5372] [General]
On [this guide part](https://textual.textualize.io/guide/layout/#layers)

it says: `An example layers declaration looks like: layers: one two three;. To add a widget to the topmost layer in this case, you'd add a declaration of layer: three; to it.`

The question is more about, the declaration of layers one two three is not shown in the example for layers.
Or have i misunderstood the text?

**Comment by TomJGooding** [comment]
The example in the description is unrelated to the code example. 

The layers are defined in [layers.tcss](https://textual.textualize.io/guide/layout/#__tabbed_17_3) -  perhaps that should be clarified?

**Comment by GronbergInfo** [comment]
Maybe it could be re-written a bit to be little more clearer.

the one two three, i assume its meant to be on the screen part of the layers.tcss
```
Screen {
    align: center middle;
    layers: below above;
}
```
